config/provisioners/in-memory-channel/in-memory-channel.yaml

```diff
@@ -193,13 +193,9 @@ spec:
       serviceAccountName: in-memory-channel-dispatcher
       containers:
         - name: dispatcher
-          image: github.com/knative/eventing/cmd/fanoutsidecar
-          args:
-            - --sidecar_port=8080
-            - --channel_provisioner=in-memory
-            - --channel_provisioner=in-memory-channel
+          image: github.com/knative/eventing/cmd/in_memory/dispatcher
           env:
           - name: SYSTEM_NAMESPACE
             valueFrom:
               fieldRef:
-                fieldPath: metadata.namespace
+                fieldPath: metadata.namespace
```

Comment:
nit: need a newline.

Reply:
Is it needed? It is the last line and I tried deploying using ko and it didn't complain.